### PR TITLE
[Iscturkey.org] New ruleset

### PR DESCRIPTION
**Pull request by agyild**

### CODE REVIEW

File: src/chrome/content/rules/Iscturkey.org.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="ISCTurkey">
```
Comment (Bisaloo) [inline src/chrome/content/rules/Iscturkey.org.xml]:
```suggestion
<ruleset name="ISCTurkey.org">
```

File: src/chrome/content/rules/Iscturkey.org.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+	<target host="iscturkey.org" />
+	<target host="www.iscturkey.org" />
+
+    <rule from="^http:" to="https:" />
+</ruleset>
```
Comment (Bisaloo) [inline src/chrome/content/rules/Iscturkey.org.xml]: Can you make sure your file ends with a newline please?

**Comment by Bisaloo** [comment]
Sorry, one more thing that I missed in my first review: you mixing spaces and tabs, could you please switch to tabs only (see L5)?

**Comment by agyild** [comment]
> 
> 
> Sorry, one more thing that I missed in my first review: you're mixing spaces and tabs, could you please switch to tabs only (see L5)?

Oops, probably mixed up when I was copy-pasting some rules as a boilerplate. Can you check again now?

**Comment by Bisaloo** [comment]
Yes, perfect! Thank you!

**Pull request merged**